manifests/config/passenger/fragment.pp

```diff
@@ -7,23 +7,25 @@
 #  $ssl_content:: content of the ssl virtual host fragment
 #
 define foreman::config::passenger::fragment(
-  $content='',
-  $ssl_content='',
+  $content=undef,
+  $ssl_content=undef,
 ) {
   require foreman::config::passenger
 
-  file { "${apache::confd_dir}/05-foreman.d/${name}.conf":
-    ensure  => present,
-    content => $content,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    notify  => Class['apache::service'],
+  if $content {
```

Comment:
I was hesitant to do this because I thought you could no longer wipe them, but I now remember that puppetlabs-apache purges unmanaged files so it should be fine, right?